core/controller/src/main/scala/whisk/core/controller/Triggers.scala

```diff
@@ -173,16 +174,21 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
                                             Path.SingleSlash + rule.action.name.asString
                                         }
                                     }.toString
-
                                     val actionUrl = Path("/api/v1") / "namespaces" / actionNamespace / "actions"
+                                    val request = HttpRequest(
+                                        method = POST,
+                                        uri = url.withPath(actionUrl + actionPath),
+                                        headers = List(Authorization(BasicHttpCredentials(user.authkey.uuid.toString, user.authkey.key.toString))),
+                                        entity = HttpEntity(MediaTypes.`application/json`, args.get.compactPrint)
```

Comment:
Two lines up toString should be asString (in other places to - all our types have asString converters; toString used for printing primarily).